inst/assumptions.m

```diff
@@ -112,7 +112,7 @@
       'if d == {}:'
       '    astr = ""'
       'elif all(d.values()):'  % all True so list them
-      '    astr = ", ".join([str(i) for i in d.keys()])'
+      '    astr = ", ".join(sorted([str(i) for i in d.keys()]))'
```

Comment:
I too looked some places and it seems there's no standard way to do it in python. So, this change looks good.. :) :+1: